src/styles/mailchimp.scss

```
@@ -7,35 +7,29 @@ $mailchimp-color-white: #fff;
 #mc_embed_signup input[type="text"] {
   font-size: 14px;
   font-weight: 400;
-  line-height: 27px;
+  line-height: 32px;
   color: $mailchimp-color-grey;
   outline: 0;
   border-width: 2px;
-  width: 400px;
-  padding-top: 7px; /* aligns input text in element */
+  width: 390px;
+  /*padding-top: 7px;  aligns input text in element */
```

Single-line comments should just use `//`

```suggestion
  // padding-top: 7px;  aligns input text in element
```